### PR TITLE
Feature add account

### PR DESCRIPTION
**Pull request by trandaison**
@at-DungNV  @at-anhnt94  Please take a look at this one and tell me if there's some where I need to fix.

@at-dungnh  please review and merge this branch. Massive thanks!

### CODE REVIEW

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\UserRequest $request User request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(UserRequest $request)
+    {
+        $user = new User($request->all());
+        $user->save();
```
Comment (dzungnguyenh) [inline app/Http/Controllers/UserController.php]: [rank B] What's happen if user data cannot save?

File: public/js/validator.custom.js [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+
+var validator = (function(){
+	var message, tests, checkField, validate, mark, unmark, field, minmax, defaults,
+		validateWords, lengthRange, lengthLimit, pattern, alertTxt, data,
+		email_illegalChars = /[\(\)\<\>\,\;\:\\\/\"\[\]]/,
+		email_filter = /^.+@.+\..{2,3}$/;
+
+	/* general text messages
+	*/
+	message = {
```
Comment (dzungnguyenh) [inline public/js/validator.custom.js]: I have a question there. This block did custom by yourself, didn't you?

File: resources/lang/en/errors.php [hunk 3]
```diff
@@ -3,4 +3,13 @@
 return [
     '404_title' => '404 Page Not Found',
     '404_error_message' => 'Sorry but this page page you are looking for does not exist.',
+
+    'field_required' => 'This field is required.',
+    '6_charactors_minimum' => '6 characters minumum.',
```
Comment (dzungnguyenh) [inline resources/lang/en/errors.php]: [rank A] please check spelling (for key and content)

File: resources/lang/en/errors.php [hunk 4]
```diff
@@ -3,4 +3,13 @@
...
+    'select_option' => 'Please select an option.',
+    'invalid_email' => 'Email address is invalid.',
+    'passwords_not_match' => 'Passwords do not match.',
```
Comment (dzungnguyenh) [inline resources/lang/en/errors.php]: [rank A] Passwords?

**Comment by dzungnguyenh** [comment]
I think almost is OK. Please check a few my comments

File: public/js/validator.custom.js (hunk 2 above)
Comment (trandaison) [inline public/js/validator.custom.js]: @at-dungnh 
Yes, this is Gentelella package's validator, I just changed a little bit validate rules and customized some error messages.

**Comment by trandaison** [comment]
@at-dungnh I have fix these issues:
- Fix language spelling.
- Fix logic execution when creating user account.
  Please review and merge this. Many thanks!

File: public/js/validator.custom.js [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+		invalid			: 'invalid input',
+		empty			: document.getElementById("empty").innerHTML,
```
Comment (dzungnguyenh) [inline public/js/validator.custom.js]: it's OK, so why don't make these message in language files and get it by jQuery selector methods?

**Comment by trandaison** [comment]
@at-dungnh 
Please review and merge this one. Thank you!

File: public/js/validator.custom.js [hunk 6]
```diff
@@ -0,0 +1,375 @@
...
+		validateWords, lengthRange, lengthLimit, pattern, alertTxt, data,
+		email_illegalChars = /[\(\)\<\>\,\;\:\\\/\"\[\]]/,
+		email_filter = /^.+@.+\..{2,3}$/;
+
+	/* general text messages
+	*/
+	message = {
+		invalid			: 'invalid input',
+		empty			: errorMessages.field_required,
+		min				: 'input is too short',
```
Comment (dzungnguyenh) [inline public/js/validator.custom.js]: [rank C] you just fixed some messages. How do about other?

**Comment by dzungnguyenh** [comment]
Perfect! Thank you.

**Pull request merged**